figshare/Article.py

```diff
@@ -677,13 +678,23 @@ def find_matched_articles(self, articles):
                             no_matched += 1
                             self.article_match_info[i] = f"article {data['id']} {version_no} ----- {data['author_dir']}"
                         else:
-                            self.article_match_info[i] = f"article {data['id']} {version_no} ----- "
+                            self.article_non_match_info[i] = f"article {data['id']} {version_no}"
 
-        # log articles id, version and dir name if matched.
-        for index in self.article_match_info:
-            self.logs.write_log_in_file('info', self.article_match_info[index], True)
+        if (self.article_match_info):
+            self.logs.write_log_in_file('info', "Curation folder found for below articles", True)
+            # log articles id, version and dir name if matched.
+            for index in self.article_match_info:
+                self.logs.write_log_in_file('info', self.article_match_info[index], True)
+
+        if (self.article_non_match_info):
+            self.logs.write_log_in_file('Warning', "Curation folder not found for below articles", True)
```

Comment:
Should the log type be `'warning'` instead of `'Warning'`? See https://github.com/UAL-RE/ReBACH/blob/1347ab0de2d1fd2e7d981da584f9bdcb135654b3/Log.py#L44

Reply:
Updated